### ISSUE TITLE
[PROPOSAL] Issue 1799 USDC from the treasury for updating socials

### ISSUE THREAD

**Issue by rockooor** [saber-hq/governance#43]
## Summary

Discussion here: https://discord.com/channels/835104494421016586/1220343074253705296/1220343074253705296

As previously discussed multiple times, we are aware that some token data & project links are outdated across various DEX analysis platforms. In order to sort this issue, I propose below plan and budget to update these information so that all token metrics, as well as associated social links such as Discord and Twitter are getting correctly updated.

### Proposed Action:

The proposal entails updating token details and social links on the following platforms:

Dexscreener:  https://dexscreener.com/
i.e.: https://dexscreener.com/solana/hiyggjp2fn53jw46e5uuskqnp3hh98jcerxegvoerwnw

Birdeye: https://birdeye.so/
i.e.: https://birdeye.so/token/Saber2gLauYim4Mvftnrasomsv6NvAuncvMEZwcLpD1?chain=solana

Dextools: https://www.dextools.io/
i.e.: https://www.dextools.io/app/en/solana/pair-explorer/HiYggjP2fN53Jw46e5UuskqNP3HH98jceRxEgVoeRwNw?t=1711016685265

### Impact:

Upon completion of these updates, all token details and associated social links will be correct. This will help to provide an accurate overview of the SBR token and will make it easier for interested users to find the correct social media profiles and all other Saber DAO related links.

Costs ### Overview:

The expenses for updating information on the mentioned platforms are as follows:

Dexscreener: $299
Birdeye: $300
Dextools: 6 SOL (current exchange rate: 1 SOL = $187,  approx. $1122. To accommodate potential fluctuations of the  SOL price, I added a little buffer; amount = $1200).

Total: $1799

### Timeline:

Once all links are available, I will promptly submit the necessary updates to the above mentioned platforms. Typically, these updates are processed within 3-5 business days. Will update here once live.

### Wallet:

2s9mUBkwftyocsRsjAzzMuZaM5f9YWn9mSDM6TQHrvKF

### TLDR

I am asking the DAO to send the amount of 1799 USDC from the Saber DAO treasury to my personal wallet to pay the update-services from above mentioned platforms.
